packages/components/src/styles/copyright/copyright.stories.ts

```diff
@@ -29,17 +29,9 @@ export default {
 };
 
 export const Default = {
-  parameters: {
-    controls: {
-      disable: true
-    }
-  },
-  render: (args: { [k: string]: any }) => {
-    return generateTemplate({
-      options: {
-        templateContent: `<div class="%CLASSES% max-w-xl" style="--copyright: '${args['--copyright']}';">%SLOT%</div>`
-      },
-      args
-    });
-  }
+  name: 'Default',
+  render: () =>
+    html`<div class="sd-copyright max-w-xl" style="--copyright: '© 2024 Solid Design System';">
+      <img src="./placeholders/images/generic.jpg" alt="A generic placeholder jpg" class="aspect-video object-cover" />
+    </div>`
```

Comment:
First story has to be interactive.

Reply:
![CleanShot 2024-08-19 at 07 07 44@2x](https://github.com/user-attachments/assets/306bf312-722f-4c25-8e4f-70237ce75c1a)

Currently broken.